api4/file_test.go

```diff
@@ -251,7 +245,7 @@ func TestUploadFiles(t *testing.T) {
 		{
 			title:                 "Happy invalid image",
 			names:                 []string{"testgif.gif"},
-			blobs:                 [][]byte{fileBytes("test-search.md")},
+			blobs:                 [][]byte{fileBytes(t,"test-search.md")},
```

Comment:
nit: it looks like we could use an extra space on a couple of these lines
```suggestion
			blobs:                 [][]byte{fileBytes(t, "test-search.md")},
```